image/templates/helm/stackrox-central/internal/config-shape.yaml

```diff
@@ -86,6 +86,15 @@ central:
       minConns: null # int
       maxConns: null # int
       statementTimeoutMs: null #int
+    config:
```

Comment:
Hmm, is this `config` nesting something we need / does it make things clearer for users?
I mean, this whole file/section/subsection is a `config`.